RocketChat/Rocket.Chat
# feat: Omnichannel Verification Visitor

## Proposed changes (including videos or screenshots)
Implemented visitor's email update, OTP generation, and verification

https://github.com/RocketChat/Rocket.Chat/assets/67470541/c6504c72-7d7e-49df-9041-c117ac43b486







## Issue(s)


## Steps to test or reproduce

1. Go to omnichannel, join an open room
2. Click on verify user button from quick action buttons.

## Further comments
